third_party/java/java_tools/README.md

```diff
@@ -17,3 +17,8 @@ third_party/java/java_tools/JacocoCoverage_jarjar_deploy.jar
 third_party/java/java_tools/turbine_deploy.jar
 third_party/java/java_tools/turbine_direct_binary_deploy.jar
 third_party/java/java_tools/SingleJar_deploy.jar
+
+The following tools were built with bazel 0.21.0 at commit 7f97ed4df69c2d9358a063cc7bb310c615ff6496 by running:
```

Comment:
nit: it's nicer to build all the tools at the same commit, but it's OK to do that once all of them are moved to checked-in deploy jars.